test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -937,6 +937,53 @@ public void HubsCannotHaveOverloadedMethods()
             }
         }
 
+        [Theory]
+        [MemberData(nameof(HubTypes))]
+        public async Task SendToMultipleClients(Type hubType)
+        {
+            dynamic endPoint = HubEndPointTestUtils.GetHubEndpoint(hubType);
+
+            using (var firstClient = new TestClient())
+            using (var secondClient = new TestClient())
+            using (var thirdClient = new TestClient())
+            {
+                Task firstEndPointTask = endPoint.OnConnectedAsync(firstClient.Connection);
```

Comment:
nit: var :trollface:

Reply:
Turns out these actually can't be var because we need `endPoint` to be dynamic. The endpoint tasks have to be explicitly typed as tasks

Reply:
You're not wrong, I just dislike anything `dynamic` :trollface:

Reply:
Might be worth refactoring these into separate tests with a common helper and using real generics (instead of Theories) sometime.